ci/test/pipeline.template.yml

```diff
@@ -114,6 +114,16 @@ steps:
       queue: linux-x86_64
     coverage: skip
 
+  - id: lint-protobuf
```

Comment:
Should this be a separate step? Does it take long? I assumed this would go into the rest of the lint jobs.

Reply:
No, it does not take long. Should I include it in `lint-fast`? However, this will collide with https://github.com/MaterializeInc/materialize/pull/19157#discussion_r1189477352 (running the check only on PRs).

Reply:
If it's fast anyway, I'd prefer it in `lint-fast`. But I haven't added linters yet, so might want to wait for a more experienced opinion.

Reply:
✅ https://github.com/MaterializeInc/materialize/pull/19157/commits/930f87a20a31a4da90c2678e3f2de59c55b6c1bd